config/ui/ui.go

```diff
@@ -22,12 +23,13 @@ var (
 
 // Config is the configuration for the UI of Gatus
 type Config struct {
-	Title       string   `yaml:"title,omitempty"`       // Title of the page
-	Description string   `yaml:"description,omitempty"` // Meta description of the page
-	Header      string   `yaml:"header,omitempty"`      // Header is the text at the top of the page
-	Logo        string   `yaml:"logo,omitempty"`        // Logo to display on the page
-	Link        string   `yaml:"link,omitempty"`        // Link to open when clicking on the logo
-	Buttons     []Button `yaml:"buttons,omitempty"`     // Buttons to display below the header
+	Title           string   `yaml:"title,omitempty"`                    // Title of the page
+	Description     string   `yaml:"description,omitempty"`              // Meta description of the page
+	Header          string   `yaml:"header,omitempty"`                   // Header is the text at the top of the page
+	Logo            string   `yaml:"logo,omitempty"`                     // Logo to display on the page
+	Link            string   `yaml:"link,omitempty"`                     // Link to open when clicking on the logo
+	Buttons         []Button `yaml:"buttons,omitempty"`                  // Buttons to display below the header
+	RefreshInterval int      `yaml:"default-refresh-interval,omitempty"` // Default Interval in seconds at which the page gets refreshed
```

Comment:
Can you rename `RefreshInterval` to `DefaultRefreshInterval` as well?

Reply:
I though about it, but there would be a naming conflict with the in-code default value:
```go
const (
    ...
    defaultRefreshInterval = 300
)
```
The only difference would be the _d_, this is confusing in my point of view.
Is there a better naming?

Reply:
variables/constants/types/methods/functions starting with a lowercase letter are not exported outside the package, so it's not too unusual imo.

The `default-refresh-interval` in the YAML is to make it more obvious for users that this value can be overriden, while the one from the constant is just for us. Honestly, you could remove the constant and set the value directly to 300 on L76 if you prefer.

Reply:
Fixed